internal/pkg/cli/pipeline_init.go

```diff
@@ -318,6 +354,35 @@ func (o *initPipelineOpts) askPipelineName() error {
 	return nil
 }
 
+func (o *initPipelineOpts) askOrValidatePipelineType() error {
+	if o.pipelineType != "" {
+		for _, typ := range pipelineTypes {
+			if o.pipelineType == typ {
+				return nil
+			}
+		}
+		return fmt.Errorf("invalid pipeline type %q; must be one of %s", o.pipelineType, english.WordSeries(utils.QuoteStringSlice(pipelineTypes), "or"))
+	}
+
+	typ, err := o.prompt.SelectOption("What type of continuous delivery pipeline is this?",
+		"A pipeline can be set up to deploy either your workloads or your environments",
+		[]prompt.Option{
+			{
+				Value: pipelineTypeWorkloads,
+				Hint:  "Deploy the workloads in your workspace",
```

Comment:
nit: We can use slightly different wording in the hint
```suggestion
				Hint:  "Deploy the services or jobs in your workspace",
```